src/env/make_env.py

```diff
@@ -0,0 +1,102 @@
+import os
+from typing import Any, List, Optional
+
+import gym
+import numpy as np
+from pynktrombonegym.wrappers.action_by_acceleration import \
+    ActionByAcceleration
+from pynktrombonegym.wrappers.log1p_mel_spectrogram import Log1pMelSpectrogram
+
+from .array_action import ArrayAction
+from .array_voc_state import ArrayVocState
+from .normalize_action_range import NormalizeActionRange
+
+
+def make_env(dataset_dirs: List[Any], file_exts: Optional[List[str]] = None, 
+             action_scaler: Optional[float] = None, low: Optional[float] = None, high: Optional[float] = None, 
+             sample_rate: int = 44100, n_mels: int = 80, dtype: Any = np.float32) -> gym.Env:
```

Comment:
Noneをデフォルトの値にすることは避けましょう。どうしてもデフォルト値に内部処理が必要な値のみ使用しましょう。(NULLやnullptrにより数々の不具合が引き起こされてきました。)  
docstringの修正もお願いします。
```suggestion
def make_env(dataset_dirs: List[Any], file_exts: List[str] = ["wav"], 
             action_scaler: Optional[float] = None, low: float = -1.0, high: float = 1.0, 
             sample_rate: int = 44100, n_mels: int = 80, dtype: Any = np.float32) -> gym.Env:
```